tests/test_parameter.py

```diff
@@ -17,7 +17,7 @@ def test_Parameter_initialization(parameter_kwargs_k3f):
     assert k3f.unit == parameter_kwargs_k3f["unit"]
     assert k3f.quantity == parameter_kwargs_k3f["value"] * parameter_kwargs_k3f["unit"]
 
-    assert k3f.type == "constant"
+    assert k3f.type == smart.model_assembly.ParameterType.constant
```

Comment:
Here is an example were it would be nice to allow strings instead. IMO it is a bit cumbersome to use the enum type here.